shiny/workflowPlot/server_files/model_plots_server.R

```diff
@@ -34,11 +34,11 @@ observeEvent(input$var_name_model, {
 
 # Check that new units are parsible and can be used for conversion
 observeEvent(input$units_model,{
-  parseable <- ifelse(udunits2::ud.is.parseable(input$units_model), "can", "cannot")
-  if(udunits2::ud.is.parseable(input$units_model)){
+  parseable <- ifelse(PEcAn.utils::unit_is_parseable(input$units_model), "can", "cannot")
```

Comment:
This line seems unused, so I think we can delete it:
```suggestion
```

Reply:
(I went ahead and committed this so that the final round of checks could kick off)